assets/css/src/common.styl

```diff
@@ -2,6 +2,12 @@
 .clearfix
   clearfix()
 
+.float-right
```

Comment:
I see you're trying to use the Tailwind approach, but I'd suggest to maintain the direction current codebase is set up in, and that is specific styling for semantic elements, instead of `style="float:right;"` equivalents.